src/sdk/index.ts

```diff
@@ -679,6 +679,7 @@ export type ProfileQuery = {
     company?: string | null;
     use_case?: string | null;
     is_authenticated: boolean;
+    is_staff?: boolean | null;
```

Comment:
i think this file is auto-generated, can you try running `yarn generate`?

(also it might already exist already on User? i see
```ts
is_staff?: Maybe<Scalars['Boolean']>;
```

Reply:
I'm not sure what User is vs ProfileQuery though

Reply:
Oooh, yup its auto generated.  
`yarn generate` removes that and changes a bunch of other things in that file.

Reply:
oh figured it out, add your "is_staff" line to `src/sdk/documents/profileQuery.graphql` then run generate!